lib/command-helpers/element-utils/common/index.js

```diff
@@ -79,6 +81,36 @@ module.exports = class CommonUtils {
         };
     }
 
+    async calcWebViewCoords(browser, {bodyWidth, pixelRatio = 1} = {}) {
+        if (
+            this._nativeElementsSize &&
+            this._nativeElementsSize.topToolbarHeight &&
+            this._nativeElementsSize.bottomToolbarHeight &&
+            this._nativeElementsSize.webviewHeight &&
+            this._nativeElementsSize.webviewWidth
```

Comment:
Why you check every option like that? Looks like in you `validate` you can just check that `this._nativeElementsSize` is not empty.